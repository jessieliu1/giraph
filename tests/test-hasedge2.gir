int main() {
	graph<int> g = [A:3 -- C:4 -- D:5 -- B:8];
	wedigraph<int> wd = [A -{7}-> C -{3}-> B];

	for_edge(e : g) {
		if (wd.has_edge(e.from(), e.to())) {
			prints("yup");
		} else {
			prints("nope");
		}
	}

	for_edge(e : wd) {
		if (g.has_edge(e.from(), e.to())) {
			prints("yup");
		} else {
			prints("nope");
		}
	}

	return 0;
}