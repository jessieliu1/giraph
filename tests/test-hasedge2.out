yup
nope
nope
yup
nope
